go.sum

```diff
@@ -2085,8 +2087,8 @@ github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2/go.mod h1:hkRG
 github.com/syndtr/goleveldb v1.0.0/go.mod h1:ZVVdQEZoIme9iO1Ch2Jdy24qqXrMMOU6lpPAyBWyWuQ=
 github.com/tarm/serial v0.0.0-20180830185346-98f6abe2eb07/go.mod h1:kDXzergiv9cbyO7IOYJZWg1U88JhDg3PB6klq9Hg2pA=
 github.com/tchap/go-patricia v2.3.0+incompatible/go.mod h1:bmLyhP68RS6kStMGxByiQ23RP/odRBOTVjwp2cDyi6I=
-github.com/terraform-providers/terraform-provider-aws v1.60.1-0.20200214022145-f0f304894df6 h1:4BVmiwuOt+ObvbVS/n+Wts7kxvxJAqXe8Z9a3ImT6II=
-github.com/terraform-providers/terraform-provider-aws v1.60.1-0.20200214022145-f0f304894df6/go.mod h1:4GMlQEMMe4rxx095oUo0OUAppV/O4GVHAvPkgaMagxY=
+github.com/terraform-providers/terraform-provider-aws v1.60.1-0.20200319153356-a183662640d9 h1:xqOfDQLPKE6UkJi4sq2/R0e/zpQQDypUzuCNPbbcXSE=
```

Comment:
I think this is my lack of understanding `gomod` but even after reading https://golang.org/cmd/go/#hdr-Pseudo_versions I don't understand why this has prefix `v1.60-1.0-` and is not simply `v2.54.0` as the commit is part of a release?

Reply:
any `v2.*.*` or greater requires module to have a `/v2` (or corresponding) suffix.. since tf aws provider is using `v2` tags but no module the go mod fails... 

so afaik using commit as target like `go mod edit -require github.com/terraform-providers/terraform-provider-aws@commit-sha` makes the go mod to change that commit to a psuedo verion based on latest v1 tag..